skel/README.md

```diff
@@ -4,16 +4,36 @@ This application was generated with [sonic](https://github.com/openware/sonic)
 
 # How to run sonic with client
 
-1. Put your frontend application to the client folder
-
-2. Run makefile
+1. Run makefile
 ```
 make asset
 ```
-It will run build in the client folder and then all build files will be moved to public/assets.
-#### **Be sure that you build your client (frontend) to the build folder, if it's another folder you can update your client (frontend) or makefile**
+It will run build in the web folder and then all build files will be moved to public/assets.
+#### **Be sure that you build your web (frontend) to the build folder, if it's another folder you can update your client (frontend) or makefile**
+
+2. Start db, redis and vault
+```
+docker-compose -f config/backend.yml up -Vd
+```
+
+3. Create db
```

Comment:
No need, because `opendax_development ` db exists by default due to compose file